prov/util/src/util_main.c

```diff
@@ -169,16 +169,10 @@ int util_getinfo(const struct util_prov *util_prov, uint32_t version,
 		return -FI_EINVAL;
 	}
 
-	ret = ofi_check_info(util_prov, version, hints);
+	ret = ofi_check_info_ex(util_prov, version, hints, info);
```

Comment:
This is now allowing a list of info structures to be returned, but the code further down in the function does not handle a list.